ospatch/updates.go

```diff
@@ -0,0 +1,116 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package ospatch
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+const (
+	rpmQuery = "/usr/bin/rpmquery"
+)
+
+func exists(path string) (bool, error) {
```

Comment:
Should we have a central place for `os` wrappers like this? I think SWR has some too?

Reply:
Maybe in the future if we have a lot of duplicate code, I prefer to keep simple functions like this close to the code that uses them.